src/components/EditImageModal/index.jsx

```diff
@@ -551,16 +613,32 @@ export default class EditImageModal extends React.Component {
           <h3>Select a previously uploaded image</h3>
         </div>
       </div>
+      {(this.state.assetsPageType === pageTypes.NORMAL || this.state.assetsPageType === pageTypes.NO_RESULTS) && (
+        <div className="row">
+          <div className="col-12">
+            <WrappedAssetsSearch />
+          </div>
+        </div>
+      )}
+      {this.state.assetsPageType === pageTypes.NORMAL && (
+        <div className="row">
+          <div className="col-12">
+            <WrappedAssetsResultsCount />
```

Comment:
How do you feel about putting the WrappedAssetsResultCount in the same row as WrappedAssetsSearch? I, personally, feel like there's, visually, some wasted spaced by having them on separate lines, so to speak, especially in our increasingly cramped modal.

Reply:
Doesn't look too bad to me, I think I'll do that.

![screenshot-20180503173032-1010x652](https://user-images.githubusercontent.com/1505923/39603806-b8abc3f2-4ef7-11e8-8be3-9a367622b6e5.png)